### ISSUE TITLE
Handle long lines

### ISSUE THREAD

**Issue by RusAlex** [vysheng/tg#726]
Has someone experienced a bug when you type a message longer than your window width , then telegram-cli prompt jumps higher and higher with each key stroke ? At the moment im trying to reproduce exactly .

**Comment by LukeLR** [comment]
Yes, I experienced irregular line jumps in tg-cli several times before, often in connection with long lines, but also when output occurred while entering a command. 
This might be connected with libreadline maybe?

**Comment by tspilk** [comment]
This has happened for as long as I have been using the telegram-cli. It is an inconvenience to use like that but doesn't affect the sent message. I just updated to latest this morning hoping this exact issue had been fixed but it hadn't yet. The alternate problem is if the window is too narrow it will just create many many text input lines on wrapping.

To replicate for me, I am in a chat_with_peer console for a message of length:
XXXX_XXXXXXX > 1234567890123456789012345678901234567890123456789012345

or less, the message will infinitely wrap. One more character long and it will wrap to previous line and shrink to previous line on each keystroke.

**Comment by LukeLR** [comment]
It would be interesting to know if it depends on the size of your console window.

**Comment by tspilk** [comment]
That was when I was resizing the window. Bigger than the text I showed it starts behaving like that

**Comment by michelesr** [comment]
It doesn't happen for me using the normal prompt... so a good way to type a long message is with:

`msg <peer> <msg>`

**Comment by michelesr** [comment]
Seems that the window size (or maybe only the width) change the behaviour of the prompt when handling long messages

**Comment by Custodia** [comment]
I am having problem with handling long text as well, if the text I type is longer than the width of my console it keeps printing the message I'm writing a couple of times a second.

**Comment by nico202** [comment]
Same problem here. On guake (+i3), the behaviour starts before the line end (about 3/4 of the line)

**Comment by msjoberg** [comment]
This happens to me as well, but only when I'm in the "chat_with_peer" mode.
